wolfcrypt/src/misc.c

```diff
@@ -356,9 +356,19 @@ WC_STATIC WC_INLINE byte ctMaskEq(int a, int b)
     return (~ctMaskGT(a, b)) & (~ctMaskLT(a, b));
 }
 
+WC_STATIC WC_INLINE word16 ctMask16GT(int a, int b)
+{
+    return (((word32)a - b - 1) >> 31) - 1;
+}
+
+WC_STATIC WC_INLINE word16 ctMask16LT(int a, int b)
+{
+    return (((word32)a - b - 1) >> 31) - 1;
```

Comment:
Maybe the *LT and *GT should be different as in width cases. Please see this: https://github.com/wolfSSL/wolfssl/pull/3001